kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/utils/TopicToTableResolver.java

```diff
@@ -83,6 +87,38 @@
     return matches;
   }
 
+  // package private for testing
+  static TableId getPartitionedTableName(TableId baseTableId, Date date) {
+    StringBuilder sb = new StringBuilder();
+    String baseTableName = baseTableId.table();
+    sb.append(baseTableName);
+    sb.append("$");
+    Calendar cal = Calendar.getInstance();
```

Comment:
I'm nervous that this code is going to use timezones, and we want everything to be UTC. Can you force it to use UTC?